App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
why not using the appsettings.json approach to configure the app ? I believe this is an old approach. 
read about the differences between between both approaches.